src/test/java/core/basesyntax/service/impl/TransactionProcessorImplTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.dao.InventoryDao;
+import core.basesyntax.dao.InventoryDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.FruitTransaction.OperationType;
+import core.basesyntax.strategy.HandlerStrategy;
+import core.basesyntax.strategy.handlers.BalanceHandler;
+import core.basesyntax.strategy.handlers.PurchaseHandler;
+import core.basesyntax.strategy.handlers.ReturnHandler;
+import core.basesyntax.strategy.handlers.SupplyHandler;
+import java.util.HashMap;
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class TransactionProcessorImplTest {
+    private static final HandlerStrategy strategy = new HandlerStrategy(new HashMap<>());

```

Comment:
initialize inside `setUp()`

Reply:
It throws NPE when initialization is moved into the @BeforeAll method()